metadata-transformer/src/main/java/no/unit/nva/metadata/extractors/AbstractExtractor.java

```diff
@@ -0,0 +1,27 @@
+package no.unit.nva.metadata.extractors;
+
+import no.unit.nva.model.EntityDescription;
+
+import java.util.function.Function;
+
+public final class AbstractExtractor {
```

Comment:
Can you add a small comment that this class extracts an Abstract and is not an abstract implementation of an extractor?